bin/coreos-install

```diff
@@ -344,10 +344,20 @@ if [[ -n "${CLOUDINIT}" ]]; then
 fi
 
 if [[ -n "${IGNITION}" ]]; then
+
+    if [[ "${IGNITION:4:3}" == "://" ]]; then
```

Comment:
Can we match against `http(s)?` instead? We wouldn't want to attempt this, for example, with `gs://` URLs.